src/picongpu/include/particles/ionization/byField/BSI/AlgorithmBSIEffectiveZ.hpp

```diff
@@ -26,7 +26,9 @@
 #include "particles/traits/GetEffectiveAtomicNumbers.hpp"
 #include "traits/attribute/GetChargeState.hpp"
 
-/** IONIZATION ALGORITHM for the BSI model
+/** @file AlgorithmBSIEffectiveZ.hpp
```

Comment:
This comment must be part of the class description. There is only one class in this file therefore there is no need to add a file description.

Some for the other changes from this pull request.